src/files/constants.ts

```diff
@@ -0,0 +1 @@
+export const fileApiPrefix = '/upload/files';
```

Comment:
What do you think about keeping the constants uppercase? (to stay consistent with https://github.com/strapi/client/blob/main/src/http/constants.ts)
```suggestion
export const FILE_API_PREFIX = '/upload/files';
```